src/pages/resultsView/mutation/MutationMapperStore.ts

```diff
@@ -244,9 +243,12 @@ export class MutationMapperStore {
         return new PdbChainDataStore(this.sortedMergedAlignmentData);
     }
 
+    // TODO currently not used, implement cache based on Genome Nexus endpoint data
+    // (not straightforward to implement, data structure is different from the previous one)
     @cached get pdbPositionMappingCache()
     {
-        return new PdbPositionMappingCache();
+        // return new PdbPositionMappingCache();
+        return null;
```

Comment:
do we not want to just delete this?

Reply:
Ideally we should implement a new cache for the new structure, but for now it might be better to just remove it completely.

Reply:
removed it completely for now, and added a new issue: https://github.com/cBioPortal/cbioportal/issues/4071